src/lib/holocene/api-pagination.svelte

```diff
@@ -52,6 +57,10 @@
   export let itemsKeyname = 'items';
   export let previousButtonLabel: string;
   export let nextButtonLabel: string;
+  export let filterable = false;
+  export let filterInputLabel: string = undefined;
```

Comment:
What do you think about making this ⬇️ instead?
```suggestion
  export let filterInputPlaceholder: string = undefined;
```

Reply:
I could be talked into just using `placeholder` to keep it inline with HTML.

Reply:
`<ApiPagination placeholder="whatever" />` doesn't make a ton of sense, cause it's not clear what the placeholder is for. I'll go with Laura's suggesiton.